src/libPersistence/ContractStorageTrie.h

```diff
@@ -0,0 +1,201 @@
+/*
```

Comment:
How come this new header file is never included anywhere?

Reply:
Looks like `ContractStorageTrie.h` and `ContractStorageTrie.cpp` are duplication of `ContractStorage.h` and `ContractStorage.cpp` respectively and are not used anymore.
Can just delete these files?

Reply:
yes, removing